Translated/Dialog/Marimo/ssn_015340.csv

```diff
@@ -0,0 +1,13 @@
+name_7443#0,"""Marimo"""
+ssn_015340_0001#0,"""You must be... one of the recent arrivals in Stia, yes?"""
+ssn_015340_0002#0,"""Have you been holding up well against the heat? Please remember to hydrate yourself before battle."""
+ssn_015340_0003#0,"""If you were to collapse before your enemy did, that would defeat the whole purpose."""
```

Comment:
"that would defeat the whole purpose" doesn't make sense for me given the context. Is it suppose to suggest that it defeats the purpose of coming to Stia?